cli/sign.go

```diff
@@ -60,7 +60,10 @@ func signExisting(c *cli.Context) error {
 		return cli.NewExitError(err, 1)
 	}
 	defer f.Close()
-
+	err = os.Chmod(tFile.Name(), artFileStat.Mode())
```

Comment:
Question: The ticket says mode. But it also mentions other file attributes, like `owner` for instance. I don't believe this is preserved through the `mode` call is it? :thinking:

Reply:
Ah, that is correct, I need to preserve the owner/group. But there shouldn't be any need for anything else right? I am assuming that e. g. the timestamp should be different.